config/envs/dev.py

```diff
@@ -12,8 +12,6 @@
 class DevConfig(Config):
     #  Application Config
     SECRET_KEY = "dev"
-    SESSION_COOKIE_NAME = "session_cookie"
-    AUTHENTICATOR_HOST = environ.get("AUTHENTICATOR_HOST", "")
```

Comment:
Was it intentional to delete these? Are they unused or being read from elsewhere?

Reply:
It was just a duplicate of what it inherits from Default Config so doesn't need to be here as well